firebase-perf/src/test/java/com/google/firebase/perf/FirebasePerformanceTestBase.java

```diff
@@ -25,12 +25,19 @@
 import com.google.firebase.perf.config.ConfigResolver;
 import com.google.firebase.perf.session.PerfSession;
 import com.google.firebase.perf.session.SessionManager;
+import com.google.firebase.perf.session.gauges.GaugeCounter;
 import com.google.firebase.perf.util.ImmutableBundle;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.BeforeClass;
+import org.robolectric.shadows.ShadowLog;
 import org.robolectric.shadows.ShadowPackageManager;
 
 public class FirebasePerformanceTestBase {
+  @BeforeClass
```

Comment:
Can you do it `@After` instead? Then be a good citizen to other unit tests in the future

Reply:
Done.